lib/connection/connect.js

```diff
@@ -0,0 +1,158 @@
+'use strict';
+const net = require('net');
+const tls = require('tls');
+const Connection = require('./connection');
+
+function connect(options, callback) {
+  if (options.family !== void 0) {
+    makeConnection(options.family, options, (err, socket) => {
```

Comment:
tiny nit: is there a reason the `return`s are on separate lines? Are we trying to explicitly prevent a return?

Reply:
I was inconsistent here, but it's also code copied from `connection.js`. I've been trying to keep `return` on its own line especially in cases where we're returning a function call, in order to make it very clear that execution stops _here_. I'll change these missed cases.